.gitignore

```diff
@@ -9,7 +9,7 @@ wp-content/upgrade/
 wp-content/uploads/
 wp-content/wp-cache-config.php
 wp-content/plugins/hello.php
-/public/wp-content/debug.log
```

Comment:
ok just removed it. not sure why it was there because I had `*.log` in the gitignore file